vipps-ecom-api.md

```diff
@@ -615,9 +616,9 @@ as the app-switch URL may be something like `vipps://`, which is not a valid URL
 The endpoints required by Vipps must be publicly available.
 
 URLs that start with `https://localhost` will be rejected. If you want to use
-localhost as fallback, please use `http://127.0.0.1`.
+localhost as fallback, please use `https://127.0.0.1`.
```

Comment:
this doesn't work with the sentence (and https doesn't work on localhost) 😅 we'll allow http on localhost for testing

Reply:
Ach! Thanks.